src/auth/strategies/jwt.strategy.ts

```diff
@@ -0,0 +1,20 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable } from '@nestjs/common';
+import { jwtConstants } from '../constants';
+import { AuthDto } from '../dto';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  constructor() {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      ignoreExpiration: false,
+      secretOrKey: jwtConstants.secret,
+    });
+  }
+
+  async validate(dto: AuthDto) {
+    return dto;
+  }
```

Comment:
hmm, is this unfinished? I think you should check the `jwt` token here if it's a valid one (it's not expired and signature matches)

Reply:
Don't `ignoreExpiration: false` and `secretOrKey: jwtConstants.secret` check if it's a valid one?

Reply:
Hmm, I think these are just configuration options. Let's do it this way: 

- currently you are not using this strategy anywhere so let's leave it as it is for now
- in the next tasks that you have ahead you will need to use this strategy
- once you implement endpoint using this strategy test / verify how it works (e.g. modify manually JWT token signature so it's invalid or alter expiration date)
- I think that using this strategy now the invalid token will pass, but I might be wrong